index.js

```diff
@@ -0,0 +1,182 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+
+const knexConfig = {
```

Comment:
No need to define a configuration object here bec you already have `knexfile.js`.